### DISCUSSION TITLE
Burst SNR calculation

### DISCUSSION THREAD

**Discussion by artgui** [mhostetter/gr-adsb#57] [General]
Inside framer.py, it is written that:

> NOTE: The median of a Rayleigh distributed random variable is 1.6 dB
> less than the average.  So add 1.6 dB to get a more accurate power
> SNR.

The probability density function of the Rayleigh distribution is

$f(x;\sigma )={\frac {x}{\sigma ^{2}}}e^{-x^{2}/(2\sigma ^{2})},\quad x\geq 0,$

Mean: $\sigma {\sqrt {\frac {\pi }{2}}}$

Median: $\sigma {\sqrt {2\ln(2)}}$

$20\log(\frac{\sigma {\sqrt {2\ln(2)}}}{\sigma {\sqrt {\frac {\pi }{2}}}})=-0.54 \text{dB}$

so isn't the median 0.54dB below the mean?

Best regards

**Comment by mhostetter** [comment]
The comment should refer to a squared Rayleigh distribution, or a chi2 distribution. I believe the 1.6 dB number is correct. You can verify, though.

**Reply by artgui** [reply]
You are correct, I made a complete mess 🤦
